utils/context/keys.go

```diff
@@ -51,12 +51,16 @@ const (
 	ReputationClientCTXKey CTXKey = "reputation_client"
 	// GeminiClientCTXKey - context key for the build time of code
 	GeminiClientCTXKey CTXKey = "gemini_client"
```

Comment:
We can remove this value, right?

Reply:
we still need the Client context key, as we are storing the gemini http client implementation on the context, and retrieving it within the handler.